src/cls/_ZPM/PackageManager/Server/SemVerFind.cls

```diff
@@ -16,7 +16,7 @@ Property Name As %String [ Private, ReadOnly ];
 
 Property Parameters As %String [ Private, ReadOnly ];
 
```

Comment:
Perhaps rename class to %ZPM.General.SemanticVersion.Find? I added it to the google sheet